packages/ezjsonm-lwt/ezjsonm-lwt.1.3.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "thomas@gazagnaire.org"
+authors: "Thomas Gazagnaire"
+license: "ISC"
+tags: ["org:mirage" "org:ocamllabs"]
+homepage: "https://github.com/mirage/ezjsonm"
+doc: "https://mirage.github.io/ezjsonm"
+bug-reports: "https://github.com/mirage/ezjsonm/issues"
+depends: [
+  "ocaml"
+  "ezjsonm"
+  "dune" {>= "1.0"}
+  "alcotest" {with-test & >= "0.4.0"}
+  "ppx_sexp_conv" {with-test}
```

Comment:
```suggestion
  "ppx_sexp_conv" {with-test & >= "v0.12.0"}
```
Otherwise fails with 
```
- Error: No ppx driver were found.
- Hint: Try upgrading or reinstalling ppx_driver.
```